_fossfund/11-coloroma.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Coloroma
-logo: 
+logo: coloroma.svg
```

Comment:
Hey @balajik10, as @emmairwin stated for a few of the other entries, I don't think Colorama has a logo. I just worked on #204 and couldn't find one when I looked, but if you do find one then make sure to add it under the `assets` > `images` > `fossfund` directory.  Please verify that other logos you added are present under that directory. Good luck, I hope your exams went well!